### PR TITLE
[next] Run Prettier

### PR DESCRIPTION
**Pull request by Timer**
This PR strictly formats files so that their diffs don't noise up other PRs.

### CODE REVIEW

File: packages/now-next/src/index.ts [hunk 1]
```diff
@@ -1,20 +1,3 @@
...
-  getLambdaOptionsFromFunction,
-  getNodeVersion,
-  getSpawnOptions,
-  getScriptName,
-  glob,
-  runNpmInstall,
-  runPackageJsonScript,
-  execCommand,
-  getNodeBinPath,
-} = buildUtils;
```
Comment (styfle) [inline packages/now-next/src/index.ts:16]: This is necessary for legacy support

Comment (Timer) [reply packages/now-next/src/index.ts:16]: Could you explain more? I just moved the code down a few lines—I didn't get rid of it.

Comment (styfle) [reply packages/now-next/src/index.ts:16]: :oh:

Did prettier re-order these?

Lets keep the `import buildUtils from './build-utils';` just above the deconstruction there.

Comment (Timer) [reply packages/now-next/src/index.ts:16]: This change was done by the auto import formatter, I don't think we should be trying to fight against it. It'll just get moved again in the future.

image

There's actually ESLint rules that ban JavaScript between `import`s, so I don't think we should move this back.

**Review by styfle — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**